### ISSUE TITLE
Create "regular" UVFITS for LOFAR data

### ISSUE THREAD

**Issue by kettenis** [casacore/casacore#1032]
Casacore includes support for "Astron-specific" UVFITS.  This flavour of UVFITS is written out whenever the telescope name in the OBSERVATION table is either WSRT or LOFAR.  This may make sense for te WSRT as the code in MSFitsOutputAstron class has a lot of references to WSRT specifics. For LOFAR data this gets in the way of the desire of members of the LOFAR long baseline group to use difmap on LOFAR data as difmap has some iserious issues interpreting "Astron-specific" UVFITS files. As the MSFitsOutputAstron code does not seem to be actively maintined anymore, my suggestion is to stop using this class for LOFAR data.

**Comment by tammojan** [comment]
Resolved by #1033. Thanks Mark!

**Comment by tikk3r** [comment]
So this slipped under the radar a bit, as at the time this fix and my deployment were close together, but it turns out we were not using this patch yet and we're experiencing trouble after applying it. This is under casacore 2.4.1.

If I try to convert a LOFAR MS to a "regular" UVFITS, ms2uvfits segfaults, but if I use the "Astron-specific"  format it doesn't. Is there a fundamental difference how each one is handled? They are built using otherwise exactly the same container environment and exactly the same build settings.

**Output using "regular" UVFITS**
```
Singularity> ./msfits/apps/ms2uvfits in=../../lbcs_calibrator_10SB.phased.nodysco.ms/ writesyscal=F
./msfits/apps/ms2uvfits: Version 20080124APS
The input MeasurementSet is: /home/frits/software/lofar-grid-hpccloud/test_difmap_newimage/lbcs_calibrator_10SB.phased.nodysco.ms
2020-07-27 13:46:34	INFO	MSFitsOutput::write	 nchan=-1 startchan=-1 stepchan=-1 avgchan=1
2020-07-27 13:46:34	INFO	MSFitsOutput::write	Converting MeasurementSet /home/frits/software/lofar-grid-hpccloud/test_difmap_newimage/lbcs_calibrator_10SB.phased.nodysco.ms to FITS file '../../lbcs_calibrator_10SB.phased.nodysco.ms/.UVF'
2020-07-27 13:46:34	INFO	MSFitsOutput::_writeMain	Writing DATA column
2020-07-27 13:46:34	INFO	MSFitsOutput::_writeMain	Frequency reference frame is TOPOCENT
....10....20....30....40....50....60....70....80....90....100%
2020-07-27 13:46:40	INFO	MSFitsOutput::write	Writing AIPS FQ table
2020-07-27 13:46:40	INFO	MSFitsOutput::writeFQ	Found 1 spectral windows 
Segmentation fault (core dumped)
```

The limited output of a backtrace in gdb is
```
2020-07-27 13:23:58	INFO	MSFitsOutput::write	Writing AIPS FQ table
2020-07-27 13:23:58	INFO	MSFitsOutput::writeFQ	Found 1 spectral windows 

Thread 1 "ms2uvfits" received signal SIGSEGV, Segmentation fault.
0x00007ffff7b2d489 in casa::MSFitsOutput::writeFQ(casa::FitsOutput*, casa::MeasurementSet const&, casa::Block<int> const&, int, double, int, double, bool, int, int, int, int) () from /opt/lofar/casacore/lib/libcasa_msfits.so.2
Missing separate debuginfos, use: dnf debuginfo-install fftw-libs-double-3.3.5-8.fc27.x86_64 fftw-libs-single-3.3.5-8.fc27.x86_64 hdf5-1.8.18-11.fc27.x86_64 libaec-1.0.2-1.fc27.x86_64 libgcc-7.3.1-6.fc27.x86_64 libgfortran-7.3.1-6.fc27.x86_64 libgomp-7.3.1-6.fc27.x86_64 libquadmath-7.3.1-6.fc27.x86_64 libstdc++-7.3.1-6.fc27.x86_64 ncurses-libs-6.0-14.20170722.fc27.x86_64 openblas-serial-0.3.2-3.fc27.x86_64 readline-7.0-7.fc27.x86_64 zlib-1.2.11-4.fc27.x86_64
(gdb) bt
#0  0x00007ffff7b2d489 in casa::MSFitsOutput::writeFQ(casa::FitsOutput*, casa::MeasurementSet const&, casa::Block<int> const&, int, double, int, double, bool, int, int, int, int) () from /opt/lofar/casacore/lib/libcasa_msfits.so.2
#1  0x00007ffff7b4600e in casa::MSFitsOutput::write() const () from /opt/lofar/casacore/lib/libcasa_msfits.so.2
#2  0x00007ffff7b46cee in casa::MSFitsOutput::writeFitsFile(casa::String const&, casa::MeasurementSet const&, casa::String const&, int, int, int, bool, bool, bool, bool, double, bool, int, unsigned int, bool) ()
   from /opt/lofar/casacore/lib/libcasa_msfits.so.2
#3  0x00000000004030d3 in main ()

```
**Output using "Astron-specific" UVFITS**
```
Singularity> ./msfits/apps/ms2uvfits in=../../lbcs_calibrator_10SB.phased.nodysco.ms/ writesyscal=F
./msfits/apps/ms2uvfits: Version 20080124APS
The input MeasurementSet is: /home/frits/software/lofar-grid-hpccloud/test_difmap_newimage/lbcs_calibrator_10SB.phased.nodysco.ms
2020-07-27 21:31:34	INFO	MSFitsOutputAstron::writeFitsFile	Converting MeasurementSet /home/frits/software/lofar-grid-hpccloud/test_difmap_newimage/lbcs_calibrator_10SB.phased.nodysco.ms to FITS file '../../lbcs_calibrator_10SB.phased.nodysco.ms.UVF'
2020-07-27 21:31:34	INFO	MSFitsOutputAstron::writeMain	Writing DATA column
2020-07-27 21:31:40	INFO	MSFitsOutputAstron::writeFitsFile	Writing AIPS FQ table
2020-07-27 21:31:40	INFO	MSFitsOutputAstron::writeFQ	Found 1 spectral windows 
2020-07-27 21:31:40	INFO	MSFitsOutputAstron::writeFitsFile	Writing AIPS AN table
2020-07-27 21:31:40	INFO	MSFitsOutputAstron::writeAN	Found 76 antennas in array #1
2020-07-27 21:31:40	INFO	MSFitsOutputAstron::writeFitsFile	Writing AIPS SU table
2020-07-27 21:31:40	INFO	MSFitsOutputAstron::writeSU	Not setting velocity types
2020-07-27 21:31:40	INFO	MSFitsOutputAstron::writeSU	writing 1 sources
ms2uvfits normally ended
```
